src/chunks/shared/messages-header-chunk/MessagesHeaderChunk.tsx

```diff
@@ -0,0 +1,91 @@
+"use client";
+import React from "react";
+import { MessagesHeaderChunkProps } from "./MessagesHeaderChunk.types";
+import Nomad from "public/images/Nomad.svg";
+import {
+  ArrowDown,
+  MobileHamburger,
+  Notification,
+  Tosin,
+} from "@assets/images";
+import Link from "next/link";
+import Image from "next/image";
+import DropDownController from "@/components/controller/drop-down-controller/DropDownController";
+// import IC_Search from "@assets/icons/IC_Search";
+
+const MessagesHeaderChunk = (props: MessagesHeaderChunkProps) => {
```

Comment:
what's this message header chunk for?

Reply:
That is where the header for the messages tab is